modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -515,36 +515,35 @@
        (when (.next rset)
          (.getString rset 1))))))
 
-;; don't redef if already definied -- test extensions override this impl
-(when-not (get (methods sql-jdbc.sync/excluded-schemas) :oracle)
```

Comment:
test extensions stopped overriding this once we switched to testing against a Docker image rather than a cloud DB